core/trino-main/src/test/java/io/trino/sql/planner/TestConnectorExpressionTranslator.java

```diff
@@ -492,6 +496,50 @@ public void testTranslateIn()
                 Optional.empty());
     }
 
+    @Test
+    public void testTranslateSimpleCase()
+    {
+        double doubleValue = 50.0d;
+        String strValue1 = "abc";
+        String strValue2 = "def";
+        Expression operand = new SymbolReference("double_symbol_1");
+        List<WhenClause> whenClauses = ImmutableList.of(
+                new WhenClause(new SymbolReference("double_symbol_2"), new SymbolReference("varchar_symbol_1")),
+                new WhenClause(new DoubleLiteral(String.valueOf(doubleValue)), new StringLiteral(strValue1)));
+        Expression defaultValue = new StringLiteral(strValue2);
+
+        ConnectorExpression operandConnectorExpression = new Variable("double_symbol_1", DOUBLE);
+
+        List<ConnectorExpression> whenClauseConnectorExpressions = ImmutableList.of(
+                new Variable("double_symbol_2", DOUBLE),
+                new Variable("varchar_symbol_1", VARCHAR_TYPE),
+                new Constant(doubleValue, DOUBLE),
+                new Constant(Slices.wrappedBuffer(strValue1.getBytes(UTF_8)), createVarcharType(strValue1.length())));
+
+        ConnectorExpression defaultValueConnectorExpression =
```

Comment:
just `defaultValue` (we already know that this variable is of `ConnectorExpression` type so no need to repeat it)